evm/loomevm.go

```diff
@@ -164,8 +174,12 @@ func (lvm LoomVm) Create(caller loom.Address, code []byte, value *loom.BigUInt)
 			)
 		}
 
+		ethereumTxHash := types.NewContractCreation(
+			uint64(auth.Nonce(lvm.state, caller)), nil, lvm.gasLimit, big.NewInt(0), code,
```

Comment:
The gas limit for the tx is supposed to be specified by the user that sends it, eventually the `/eth` interface will allow submitting Eth txs and we can take the gas limit from there... but for now we can just set it to `math.MaxUint64`.

Reply:
Done